### PR TITLE
Clowns will now always like bananas.

### PR DESCRIPTION
**Pull request by spiders24110**
## About The Pull Request
Clown's liver makes them like bananas, ignoring their racial food preferences.

## Why It's Good For The Game
I don't think clown moths should vomit from eating bananas. They are clowns, after all.
Also clowns are healed from eating them, so it's a bit silly that they vomit from their funny medicine.

## Changelog

:cl:
balance: Non-human clowns enjoy eating bananas now.
/:cl:

### CODE REVIEW

**Review by JohnFulpWillard — APPROVED** [review]

File: code/modules/hydroponics/grown/banana.dm [hunk 1]
```diff
@@ -28,12 +28,33 @@
...
+				food_flags = food_flags,\
+				foodtypes = foodtypes,\
+				volume = max_volume,\
+				eat_time = eat_time,\
+				tastes = tastes,\
+				eatverbs = eatverbs,\
+				bite_consumption = bite_consumption,\
+				microwaved_type = microwaved_type,\
+				junkiness = junkiness,\
+				check_liked = CALLBACK(src, .proc/check_liked))
```
Comment (Mothblocks) [inline code/modules/hydroponics/grown/banana.dm:44]: Only indent once

File: code/modules/hydroponics/grown/banana.dm [hunk 2]
```diff
@@ -28,12 +28,33 @@
 	juice_results = list(/datum/reagent/consumable/banana = 0)
 	distill_reagent = /datum/reagent/consumable/ethanol/bananahonk
 
+///Override for checkliked callback
+/obj/item/food/grown/banana/MakeEdible()
+	AddComponent(/datum/component/edible,\
```
Comment (Mothblocks) [inline code/modules/hydroponics/grown/banana.dm:33]:
```suggestion
	AddComponent(
		/datum/component/edible,\
```

File: code/modules/hydroponics/grown/banana.dm (hunk 1 above)
Comment (Mothblocks) [inline code/modules/hydroponics/grown/banana.dm:44]: Then put this last parenthese on its own line, dedented one

**Review by Mothblocks — CHANGES_REQUESTED** [review]

File: code/modules/hydroponics/grown/banana.dm (hunk 1 above)
Comment (Striders13) [reply code/modules/hydroponics/grown/banana.dm]: aight, tell me if I did it wrong

**Review by Mothblocks — APPROVED** [review]
Undraft if ready

**Pull request merged**